File: order/src/main/java/localfooddelivery/domain/Ordered.java
package localfooddelivery.domain;

import localfooddelivery.domain.*;
import localfooddelivery.infra.AbstractEvent;
import java.util.*;
import lombok.*;

@Data
@ToString
public class Ordered extends AbstractEvent {

    private Long orderId;
    private String menu;
    private Integer qty;

    public Ordered(Order aggregate){
        super(aggregate);
    }
    public Ordered(){
        super();
    }
}

File: order/Ordered.py
from AbstractEvent import AbstractEvent
import json
from datetime import datetime

class Ordered(AbstractEvent):
    orderId : int
    menu : str
    qty : int
    
    def __init__(self):
        super().__init__()
        self.orderId = None
        self.menu = None
        self.qty = None

